### PR TITLE
Add code for comments

### PR DESCRIPTION
**Pull request by TahleeJ**
I have added code within my pom.xml, index.html, script.js, DataServlet.java, and style.css files to handle and display comments. In my pom.xml file I added dependencies for gson and datastore. In my index.html file I altered the body tag to call the JavaScript function handling the comments when the page loads, and I also created a comment section at the bottom of the page to display the website with accompanying styling in my style.css file. I altered both my DataServlet.java and script.js files to handle server requests and send back information as a json string. I also changed my script.js files to have more descriptive variable names.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -14,19 +14,65 @@
...
+
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private static List<String> comments = new ArrayList<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // List<String> list = new ArrayList<>(Arrays.asList(new String[]{"First message", "Second message", "Third message"}));
```
Comment (cshei) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:41]: Remove comment?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -5,22 +5,48 @@ for (i = 0; i < 3; i++) {
...
   for (i = 0; i < slides.length; i++) {
       slides[i].style.display = "none";  
   }
-  slides[slideIndices[o]-1].style.display = "block";  
-}
\ No newline at end of file
+    slides[slideIndices[slideSection]-1].style.display = "block";  
+}
+
+// async function getHelloMessage() {
```
Comment (cshei) [inline portfolio/src/main/webapp/script.js:28]: Remove comments

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -5,22 +5,48 @@ for (i = 0; i < 3; i++) {
...
+
+function getComments() {
+   fetch('/data').then(response => response.json()).then((comments) => {
+    const commentBox = document.getElementById("comment-box");
+
+    comments.forEach((comment) => {
+        commentBox.appendChild(createComment(comment));
+    })
+
+    // for (i = 0; i < comments.length; i++) {
```
Comment (cshei) [inline portfolio/src/main/webapp/script.js:42]: ditto

**Review by cshei — APPROVED** [review]

**Comment by TahleeJ** [comment]
I will remove the comments, but I left them all there for future use if I ever needed a quick reference for code I had written before. Would it be better to move them into another file within the project dedicated to those little notes to myself, or should they be put somewhere separate?

**Pull request merged**